R/driver-databricks.R

```diff
@@ -227,6 +231,15 @@ databricks_auth_args <- function(host, uid = NULL, pwd = NULL) {
     ))
   }
 
+  if (is_installed("connectcreds") && connectcreds::has_service_account_token(workspace)) {
```

Comment:
We could also just add a conditional on the connectcreds package version here, I think? And then merge without a Remotes on the dev version.